test/unit/plugin/importer_registry_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class ImporterRegistry < ActiveSupport::TestCase
+  class TestClass
+  end
+
+  def test_register
+    fact_importer = Foreman::Plugin::ImporterRegistry.new
+    fact_importer.register(:test_class, TestClass, false)
+    assert_equal fact_importer.importers[:test_class], "ImporterRegistry::TestClass"
+  end
+
+  def test_register_default
+    fact_importer = Foreman::Plugin::ImporterRegistry.new
+    fact_importer.register(:test_class, TestClass, true)
+    assert_equal fact_importer.importers[:test_class], "ImporterRegistry::TestClass"
+    assert_equal fact_importer.importers[:random_key], "ImporterRegistry::TestClass"
+  end
+
+  def test_get
+    fact_importer = Foreman::Plugin::ImporterRegistry.new
+    fact_importer.register(:test_class, TestClass, false)
+    assert_equal fact_importer.get(:test_class), TestClass
+  end
+
+  def test_get_default
+    fact_importer = Foreman::Plugin::ImporterRegistry.new
+    fact_importer.register(:test_class, TestClass, true)
+    assert_equal fact_importer.get(:random_key), TestClass
+  end
+end
```

Comment:
Can you please add a test for `fact_features` method?
Something along the lines of
``` ruby
  def test_get_default
    fact_importer = Foreman::Plugin::ImporterRegistry.new
    fact_importer.register(:test_class, TestClass, true)
    TestClass.any_instance.expects(:authorized_smart_proxy_features).returns([:test_feature])
    assert_equal :foo, fact_importer.fact_features.first
  end
```